api/app/tests/hfi/test_hfi_endpoint_request.feature

```diff
@@ -17,3 +17,13 @@ Feature: /hfi/
             | /api/hfi-calc/     | test_hfi_endpoint_request_missing_data.json             | 200         | hfi/test_hfi_endpoint_response_missing_data.json             |
             # Test load request scenario
             | /api/hfi-calc/load | test_hfi_endpoint_request_load.json                     | 200         | hfi/test_hfi_endpoint_load_response.json                     |
+
+    Scenario: HFI - pdf download
```

Comment:
You need to update api/app/tests/hfi/test_hfi_endpoint_request.py with the scenario.
```
@pytest.mark.usefixtures('mock_jwt_decode')
@scenario('test_hfi_endpoint_request.feature', 'HFI - pdf download')
def test_pdf_scenario():
    """ BDD Scenario. """
    pass
```